docs/source/package_reference/mixins.mdx

```diff
@@ -0,0 +1,18 @@
+# Mixins & serialization methods
+
+## Mixins
+
+The `huggingface_hub` library offers a range of mixins that can be used as a parent class for your
+objects, in order to provide simple uploading and downloading functions.
+
+### PyTorch
+
+[[autodoc]] ModelHubMixin
+
+### Keras
+
+[[autodoc]] from_pretrained_keras
+
+[[autodoc]] push_to_hub_keras
+
+[[autodoc]] save_pretrained_keras
```

Comment:
I think this should also have `KerasModelHubMixin` for users using the subclassing TF API